src/robusta/integrations/webex/sender.py

```diff
@@ -0,0 +1,253 @@
+from ...core.reporting.base import Finding, FindingSeverity
+from ...core.reporting.utils import is_image, convert_svg_to_png, SVG_SUFFIX, PNG_SUFFIX
+from ...core.reporting.blocks import (
+    MarkdownBlock,
+    BaseBlock,
+    FileBlock,
+    TableBlock,
+    List,
+)
+from ...core.sinks.transformer import Transformer
+
+from PIL import Image
+from io import BytesIO
+from webexteamssdk import WebexTeamsAPI
+from fpdf import FPDF
+import os
+
+
+INVESTIGATE_ICON = "\U0001F50E"
+SILENCE_ICON = "\U0001F515"
```

Comment:
You can switch these with the actual icons , 🔕🔎 just a syntactic sugar

Reply:
Updated. Let me know if its ok or anything needs to be changed.